ArgoCDApplication.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kubescape
+spec:
+  destination:
+    name: ''
+    namespace: kubescape
+    server: 'https://kubernetes.default.svc' # change to your server
+  source:
+    path: charts/kubescape-operator
+    repoURL: 'https://github.com/kubescape/helm-charts'
+    targetRevision: argo
```

Comment:
```suggestion
    targetRevision: HEAD
```

Reply:
What is blocking you from changing this?

Reply:
I wanted to test it with Akinox this afternoon